### PR TITLE
fix(db/loot): Harbinger Skyriss Normal Loot

### PR DESCRIPTION
**Pull request by neifion-00000000**
## Changes Proposed:
- Fixes Harbringer Skyriss loot matching image below 
image
-  I propose gultask not try to replace me you got enough on your plate :P 
-  kidding <3 

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/15727
- Closes https://github.com/chromiecraft/chromiecraft/issues/5292

## SOURCE:

https://www.youtube.com/watch?v=qS7CXZzk8Qs
https://www.youtube.com/watch?v=sm7VdGKY2EA
https://www.youtube.com/watch?v=JXiFiz9Ga5w
https://www.youtube.com/watch?v=HuDJJLQd9rQ
https://www.youtube.com/watch?v=-4qHcScnR3k
https://www.youtube.com/watch?v=iyo7nt_NmXw
https://www.youtube.com/watch?v=4S67iYnePoM
https://www.youtube.com/watch?v=fLUK1Ax2zPs
https://youtu.be/6ygGxca781A
I think those are the exact sources used to solve this, plus or minus a video or two as I did have to sort out some pserver videos, I think I got rid of those properly for this list

thanks beef for the help getting this all done and your amazing eyes

## Tests Performed:

-
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by heyitsbench** [comment]
Might want to include the issues to close if applicable. 😛

File: data/sql/updates/pending_db_world/rev_1680298149569301100.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+--
+-- Harbinger Skyriss Normal Loot
+UPDATE `reference_loot_template` SET `GroupId`='3' WHERE `Entry`=25004 AND `Item` IN (28413, 28414, 28415, 28416, 28418, 28419);
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1680298149569301100.sql:3]: quotes around integers
```suggestion
UPDATE `reference_loot_template` SET `GroupId`=3 WHERE `Entry`=25004 AND `Item` IN (28413, 28414, 28415, 28416, 28418, 28419);
```

Comment (heyitsbench) [reply data/sql/updates/pending_db_world/rev_1680298149569301100.sql:3]: Quotes required for swag,

**Review by Gultask — APPROVED** [review]

**Comment by neifion-00000000** [comment]
one day I'll remember to remove all the quotes around integers HeidiSQL adds without being told, tho this one was because I copy/pasted the last one and never updated my local

**Pull request merged**